### PR TITLE
Merging Current Progress 

### PR DESCRIPTION
**Pull request by Mandersss**
### **Progress**:
- Created a base UNet structure using a reference implemented in Keras (Link in markdown) and diagram from primary research paper from UNet's creators
- Import CT Scans for use via file upload from local system
- Data loader created to hold CT Scans and Masks as images and labels

### **Next Steps**: 
- Decide on any worth while data augmentations
- Decide on ideal optimizers and loss criteria
- Create basic training loop and decide on training time 

### Issues: 
-  The image that is output from the network is reduced in pixel size, and the outputs show that it retains only an area of the original image
- Solution: Either understand how the paper interpreted the outputs OR modify decoder section and concatenations to re-generate original image size

### **Resolutions to Issues:**
- Modified the network to maintain the image dimensions so they can be compared to the provided masks. 
- Network is NOW TRAINABLE! Needs more ideal optimizer, loss function, and some additional network enhancements

### **Next Steps:**
- Consider different loss functions (Maybe some more ideal for image segmentation?)
- Look to avoid overfitting with early stopping and network weight stuff

### CODE REVIEW

**Comment by Mandersss** [comment]
Updated code to allow training loop to run
